src/test/java/fi/aalto/cs/intellij/actions/EnablePluginsActionTest.java

```diff
@@ -0,0 +1,46 @@
+package fi.aalto.cs.intellij.actions;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.ide.plugins.IdeaPluginDescriptorImpl;
+import com.intellij.notification.Notification;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.jdom.JDOMException;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class EnablePluginsActionTest {
+
+  @Test
+  public void testActionPerformed() throws IOException, JDOMException {
+    IdeaPluginDescriptorImpl ideaPluginDescriptor = getIdeaPluginDescriptor();
+    ideaPluginDescriptor.setEnabled(false);
+    List<IdeaPluginDescriptor> descriptors = new ArrayList<>();
+    descriptors.add(ideaPluginDescriptor);
+
+    assertFalse(ideaPluginDescriptor.isEnabled());
+
+    EnablePluginsAction enableMissingPluginsAction = new EnablePluginsAction("",
+        descriptors);
+    AnActionEvent anActionEvent = Mockito.mock(AnActionEvent.class);
+    Notification notification = Mockito.mock(Notification.class);
+    enableMissingPluginsAction.actionPerformed(anActionEvent, notification);
+
+    assertTrue(ideaPluginDescriptor.isEnabled());
+  }
+
+  @NotNull
+  private IdeaPluginDescriptorImpl getIdeaPluginDescriptor() throws IOException, JDOMException {
+    File filePath = new File("src/test/resources/plugins/a+/plugin.xml");
```

Comment:
**test data:** we could some dummy plugin.xml s like in here, so there won't be a need for a manual creation of a IdeaPluginDescriptor (an entity that seem to represent the plugin entity ([there is another one](https://github.com/JetBrains/intellij-community/blob/505e701fbcd34783ee8f5f01b1a4a3363154e55f/platform/platform-impl/src/com/intellij/ide/plugins/PluginNode.java), but it seems to used for a different purpose))